template-transformer/handler.lua

```diff
@@ -136,10 +136,10 @@ function TemplateTransformerHandler:body_filter(config)
       if body == nil then
         return ngx.ERROR
       end
-      local transformed_body = template_transformer.get_template(config.response_template){headers = headers,
-                                                                                           body = body,
-                                                                                           raw_body = raw_body,
-                                                                                           status = ngx.status}
+      local transformed_body = template_transformer.get_template(config.response_template, body) {headers = headers,
+                                                                                                  body = body,
+                                                                                                  raw_body = raw_body,
+                                                                                                  status = ngx.status}
```

Comment:
Podia criar um namespace utils com o cjson dentro e passar aqui com esses outros parâmetros. Eles ficam disponíveis no template.
```suggestion
                                                                                                  status = ngx.status,
                                                                                                  utils = utils}
```

Reply:
seguindo a pedidos léo implementei sua sugestão, aumentou uma linha mas ficou bem mais flexivel a manipulacao das tabelas
![Captura de Tela 2020-01-16 às 16 31 06](https://user-images.githubusercontent.com/5008825/72558925-6a630680-3882-11ea-855a-1574d8d06f5f.png)
![Captura de Tela 2020-01-16 às 16 31 14](https://user-images.githubusercontent.com/5008825/72558926-6afb9d00-3882-11ea-9929-fd8ace2ba87b.png)